src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksBatchUtilityServiceImpl.java

```diff
@@ -183,7 +183,7 @@ public void registerKfsPvenApprovalForExistingPaymentWorksVendor(String kfsVendo
                      + "' and kfsVendorDocumentNumber '" + kfsVendorDocumentNumber + "' and pmwVendorRequestId '" + pmwVendor.getPmwVendorRequestId()
                      + "' and vendor number '");
             getPaymentWorksVendorDao().updateExistingPaymentWorksVendorInStagingTable(pmwVendor.getId(), PaymentWorksConstants.KFSVendorProcessingStatus.VENDOR_APPROVED, vendorDetail.getVendorHeaderGeneratedIdentifier(),
-                                                                                      vendorDetail.getVendorDetailAssignedIdentifier(), getDateTimeService().getCurrentTimestamp());
+                                                                                      vendorDetail.getVendorDetailAssignedIdentifier(), PaymentWorksConstants.SupplierUploadStatus.PENDING_PAAT, getDateTimeService().getCurrentTimestamp());
         } else {
```

Comment:
For consistency with the way that the other status values are set, I do not believe that the SupplierUploadStatus should be set here. It is appropriately set in service class PaymentWorksNewVendorPayeeAchServiceImpl in method setStatusValuesForPaymentWorksProcessedNewVendorKfsVendorCreatedAchPendingPven and the upload back to PMW cannot happen for a vendor originating in PMW cannot happen until the PAAT batch job runs.

Reply:
Ok, I'll remove the status setup here.